lib/constants/videoState.js

```diff
@@ -1,17 +1,19 @@
+/* eslint-disable */
+
 const videoState = {
-	STOPPED: Symbol('stopped'),
-	ERROR: Symbol('error'),
-	PLAYING: Symbol('playing'),
-	PAUSED: Symbol('paused'),
-	CONNECTING: Symbol('connecting'),
-	BUFFERING: Symbol('buffering'),
-	FINISHED: Symbol('finished'),
-	IDLE: Symbol('idle'),
-	PREPARING: Symbol('preparing'),
-	PREPARED: Symbol('prepared'),
-	PLAYBACK_COMPLETED: Symbol('playback_completed'),
-	UNKNOWN: Symbol('unknown'),
-	REVERSING: Symbol('reversing'),
+	STOPPED:            'stopped',
+	ERROR:              'error',
+	PLAYING:            'playing',
+	PAUSED:             'paused',
+	CONNECTING:         'connecting',
+	BUFFERING:          'buffering',
+	FINISHED:           'finished',
+	IDLE:               'idle',
+	PREPARING:          'preparing',
+	PREPARED:           'prepared',
+	PLAYBACK_COMPLETED: 'playbackCompleted',
```

Comment:
@AntonSBilan I talked to BE, they are not willing to make comparison case-insensitive for religious reasons. Please, revert this to "playback_completed"